tests/php/sync/server/class.jetpack-sync-test-replicastore.php

```diff
@@ -167,6 +169,10 @@ function spam_comment( $comment_id ) {
 		$this->comments[ $comment_id ]->comment_approved = 'spam';
 	}
 
+	function post_trashed_comment( $comment_id ) {
```

Comment:
Rather than something as specific as post_trashed_comment, I think we should implement something a little more general like `bulk_update_comment_approved( $comment_ids, 'post-trashed' )`. Seems like in the long run that would reduce noise in the API.

Also, this function should be implemented in the replicastore interface and wp replicastore, and we need a test in `test_interface.jetpack-sync-replicastore.php`.